### PR TITLE
NGWMN-1720 added license, code, and PR metafiles to project

### PR DESCRIPTION
**Pull request by duselmann**

### CODE REVIEW

File: code.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+[
```
Comment (cschroed-usgs) [inline code.json]: The [code.json validator](https://code.gov/#!/policy-guide/docs/compliance/inventory-code/tools/validate-schema) wants the square brackets removed. After that it, should be fully valid with version 1! :tada:

Comment (cschroed-usgs) [reply code.json]: Curious: do you know if the USGS prefers code.json version 1 over version 2? I should know these things, but I don't... I haven't had the pleasure of bringing projects into compliance yet.

Comment (duselmann) [reply code.json]: Thanks for catching that issue. I am not sure about the code.json version; I based this off the file in the ngwmn-ui project.

Comment (duselmann) [reply code.json]: array brackets removed

**Pull request merged**